processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -1134,6 +1136,12 @@ public SelectQueryBuilder intervals(List<Interval> l)
       return this;
     }
 
+    public SelectQueryBuilder descending(boolean descending)
```

Comment:
Also I think in Druids and in other places, we did not follow the 'set' convention in most of the builder classes.